packages/google_ml_kit/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
"native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_face_detection","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_face_detection-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_image_labeling","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_image_labeling-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_language_id","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_language_id-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_object_detection","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_object_detection-0.4.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_pose_detection","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_pose_detection-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_selfie_segmentation","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_selfie_segmentation-0.0.1/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_smart_reply","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_smart_reply-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_text_recognition","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_text_recognition-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_translation","path":"/Users/andrew.coutts/.pub-cache/hosted/pub.dartlang.org/google_mlkit_translation-0.3.0/","native_build":true,"dependencies":["google_mlkit_commons"]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"google_mlkit_barcode_scanning","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_commons","dependencies":[]},{"name":"google_mlkit_digital_ink_recognition","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_entity_extraction","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_face_detection","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_image_labeling","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_language_id","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_object_detection","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_pose_detection","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_selfie_segmentation","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_smart_reply","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_text_recognition","dependencies":["google_mlkit_commons"]},{"name":"google_mlkit_translation","dependencies":["google_mlkit_commons"]}],"date_created":"2022-07-11 10:05:51.298252","version":"3.1.0-9.0.pre"}
```

Comment:
This is a generated file; do not edit or check into version control

Reply:
Done in 17824124197c2b19a0e63ce32244162bf15f371d